.storybook/webpack.config.js

```diff
@@ -6,7 +6,7 @@ const webpack = require.main.require('webpack');
 module.exports = storybookBaseConfig => {
 	// remove Uglification plugin to improve build time in CI
 	const uglifyIndex = storybookBaseConfig.plugins.findIndex(
-		element => element instanceof webpack.optimize.UglifyJsPlugin
+		element => element instanceof webpack.optimize.UglifyJsPlugin,
```

Comment:
The trailling comma fails with my node, and in our env too I think

Reply:
exact travis use nodejs v6. We can upgrade to the LTS 8

Reply:
node 6 is still in active LTS